EventListener/LoadORMMetadataSubscriber.php

```diff
@@ -310,6 +313,15 @@ protected function unsetFieldMappings(ClassMetadataInfo $metadata, $wasMappedSup
             // to make this work correctly, these fields will later be re-added in setFieldMappings() to the actual
             // class (but we will keep the declared / inherited flags this time)
             foreach ($metadata->fieldMappings as $name => $mapping) {
+
+                // fields are already declareds
+                if ($this->isEmbeddedProperty($name,$mapping)) {
```

Comment:
Super minor, but should be written with a space after the comma like:
  `if ($this->isEmbeddedProperty($name, $mapping)) {`

( see https://www.php-fig.org/psr/psr-2/ )